example/lib/generated/i18n.dart

```diff
@@ -11,7 +11,7 @@ class S {
  
   static const GeneratedLocalizationsDelegate delegate = GeneratedLocalizationsDelegate();
 
-  static S of(BuildContext context) {
+  static S? of(BuildContext context) {
     return Localizations.of<S>(context, S);
   }
```

Comment:
```suggestion
  static S of(BuildContext context) {
    final localization = Localizations.of<S>(context, S);
    assert(() {
      if (localization == null) {
        throw FlutterError(
            'S requested with a context that does not include S.');
      }
      return true;
    }());
    return localization!;
  }
```
I suggest that we don't return a nullable `S` in the `of` method, because
- standard libraries like `Theme` or `Navigator` don't return nullable in the `of` method and
- we don't have to write `S.of(context)!` everywhere.

Reply:
Good point, you are right. I will make the change and update de PR. Thanks!